### PR TITLE
[Notifier] Add Termii bridge

### PR DESCRIPTION
**Pull request by gnito-org**
| Q             | A
| ------------- | ---
| Branch?       | 6.3
| Bug fix?      | no
| New feature?  | yes
| Deprecations? | no
| Tickets       | 
| License       | MIT
| Doc PR        | https://github.com/symfony/symfony-docs/pull/17486
| Recipe         | https://github.com/symfony/recipes/pull/1156

This PR add the [Termii](https://www.termii.com) Notifier bridge.

### CODE REVIEW

File: src/Symfony/Component/Notifier/Bridge/Termii/.gitattributes [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/Tests export-ignore
+/phpunit.xml.dist export-ignore
+/.gitattributes export-ignore
+/.gitignore export-ignore
```
Comment (OskarStark) [inline src/Symfony/Component/Notifier/Bridge/Termii/.gitattributes]: Please add a newline to remove the red pictogram, thanks

File: src/Symfony/Component/Notifier/Bridge/Termii/.gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+vendor/
+composer.lock
+phpunit.xml
```
Comment (OskarStark) [inline src/Symfony/Component/Notifier/Bridge/Termii/.gitignore]: Please add a newline to remove the red pictogram, thanks

File: src/Symfony/Component/Notifier/Bridge/Termii/LICENSE [hunk 3]
```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2019-2022 Fabien Potencier
```
Comment (OskarStark) [inline src/Symfony/Component/Notifier/Bridge/Termii/LICENSE]:
```suggestion
Copyright (c) 2022 Fabien Potencier
```

File: src/Symfony/Component/Notifier/Bridge/Termii/README.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+```
+TERMII_DSN=termii://API_KEY@default?from=FROM&channel=CHANNEL
+```
+
+where:
+
+- `API_KEY` is your Termii API key
+- `FROM` is your sender
+- `CHANNEL` is your channel (generic, dnd, whatsapp)
+
```
Comment (OskarStark) [inline src/Symfony/Component/Notifier/Bridge/Termii/README.md]:
```suggestion
 - `API_KEY` is your Termii API key
 - `FROM` is your sender
 - `CHANNEL` is your channel (generic, dnd, whatsapp)

```

File: src/Symfony/Component/Notifier/Bridge/Termii/README.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+Resources
+---------
+
+* [Contributing](https://symfony.com/doc/current/contributing/index.html)
+* [Report issues](https://github.com/symfony/symfony/issues) and
+  [send Pull Requests](https://github.com/symfony/symfony/pulls)
+  in the [main Symfony repository](https://github.com/symfony/symfony)
```
Comment (OskarStark) [inline src/Symfony/Component/Notifier/Bridge/Termii/README.md]:
```suggestion
 * [Contributing](https://symfony.com/doc/current/contributing/index.html)
 * [Report issues](https://github.com/symfony/symfony/issues) and
   [send Pull Requests](https://github.com/symfony/symfony/pulls)
   in the [main Symfony repository](https://github.com/symfony/symfony)
```

Please check your other bridge PRs for this leading whitespace too, thanks

**Review by OskarStark — COMMENTED** [review]
I left some comments

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Regarding fabbot:
CleanShot 2022-11-30 at 17 04 02@2x

(1) and (2) are false positives, the other ones can be applied

**Comment by gnito-org** [comment]
I must be half-blind. Where do I see / access that fabbot report?

Screenshot

**Comment by OskarStark** [comment]
> I must be half-blind. Where do I see / access that fabbot report?

CleanShot 2022-11-30 at 20 15 38@2x

You need to scroll inline

**Comment by gnito-org** [comment]
My bad that I didn't see the big MORE button and wide scroll bars.

**Review by OskarStark — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @gnito-org.

**Pull request merged**